pkgs/top-level/all-packages.nix

```
@@ -12742,6 +12740,8 @@ with pkgs;
 
   rgp = libsForQt5.callPackage ../development/tools/rgp { };
 
+  ringboard-wayland = callPackage ../by-name/ri/ringboard.nix { backend = "wayland"; };
```

Has the name been changed to `displayServer`? https://github.com/NixOS/nixpkgs/pull/445583#discussion_r2377499810

```suggestion
  ringboard-wayland = callPackage ../by-name/ri/ringboard.nix { displayServer = "wayland"; };
```

https://github.com/NixOS/nixpkgs/pull/445583/files#diff-11a1ad0c1ab1361e377ba87d5793622cb6ff32393c0f4a71881f13d0f8403fc2R11

---

Good catch!